meteoritics-and-planetary-science.csl

```diff
@@ -125,8 +129,8 @@
   </citation>
   <bibliography hanging-indent="true" et-al-min="10" et-al-use-first="1">
     <sort>
-      <key macro="author"/>
-      <key variable="title"/>
+      <key macro="author-short" names-use-first="1" names-min="2"/>
+      <key macro="year-date"/>
```

Comment:
rmzelle is right, it isn't sorted quite correctly. I think this code fixes it:
`   <citation et-al-min="3" et-al-use-first="1" disambiguate-add-year-suffix="true" disambiguate-add-givenname="true" givenname-disambiguation-rule="by-cite" initialize-with="." collapse="year">
   <sort>
      <key macro="year-date"/>
      <key macro="author-short"/>
    </sort>
    <layout prefix="(" suffix=")" delimiter="; " >
      <group delimiter=", ">
        <group delimiter=" ">
	   <text macro="author-short"/>
           <text macro="year-date"/>
        </group>`

That should disambiguate if different authors have the same surname. Creates a citation like
> (Kita et al. 1998, 2000, 2005; Zhang et al. 2015; J. Zhang et al. 2016; L.-Y. Zhang et al. 2016)
instead of all the Zhang et al.'s being lumped together. This is consistent with the Chicago Style, which MaPS is branched off of. Chicago may use full first names to disambiguate, but I've never seen that in MaPS.